window.c

```diff
@@ -133,6 +134,28 @@ void win_init(win_t *win)
 	f = win_res(db, RES_CLASS ".font", "monospace-8");
 	win_init_font(e, f);
 
+	const char *title_prefix = win_res(db, RES_CLASS ".title.prefix", NULL);
+	const char *title_suffixmode = win_res(db, RES_CLASS ".title.suffixmode", NULL);
+
+	if (options->title_prefix != NULL) {
+		win->title_prefix = options->title_prefix;
+	} else if (title_prefix != NULL) {
+		win->title_prefix = title_prefix;
+	} else {
+		win->title_prefix = TITLE_PREFIX;
+	}
+
+	if (options->title_suffixmode == NULL && title_suffixmode == NULL) {
+		win->title_suffixmode = TITLE_SUFFIXMODE % SUFFIXMODE_COUNT;
+	} else {
+		char *endptr;
+		const char *nptr = options->title_suffixmode != NULL
+						 ? options->title_suffixmode : title_suffixmode;
+		win->title_suffixmode = strtol(nptr, &endptr, 10) % SUFFIXMODE_COUNT;
+		if (nptr == endptr)
+			fputs("sxiv: invalid suffixmode, assuming 0\n", stderr);
+	}
+
```

Comment:
Let's remove Xresources completely and just have config.h and cli flags. Users can use the `nsxiv-env` script to set default cli flags if they installed a binary version.